src/App.scss

```diff
@@ -37,6 +37,10 @@ body {
   color: #000;
 }
 
+.navLinks > li > a:hover {
+  color: $blue;
+}
+
 .colorBlue {
   color: $blue;
 }
```

Comment:
- [ ] You are expected to replicate the design exactly. Kindly implement other missing features as we have in the design, like the user icon and progress circle and so on.

  You can add placeholders for features that are not availiable. But design should be the same.

  Your Implementation | Zeplin Design
  :---:|:---:
  ![tugsem-style-his](https://user-images.githubusercontent.com/81468675/173089291-bf768dcb-427a-4d68-b740-099b4fe5f6ef.jpg) | ![tugsem-style](https://user-images.githubusercontent.com/81468675/173089299-660c4c8d-7ebb-4257-aa4c-da3b7f332093.jpg)